src/tests/java/htsjdk/samtools/CRAMFileWriterWithIndexTest.java

```diff
@@ -124,8 +130,9 @@ public void testUnnecessaryIO() throws IOException {
 
         CRAMFileReader reader = new CRAMFileReader(tabuIS, new ByteArraySeekableStream(indexBytes), source, ValidationStringency.SILENT);
         try {
-            reader.queryAlignmentStart(header.getSequence(refID).getSequenceName(), 1);
+            CloseableIterator<SAMRecord> it = reader.queryAlignmentStart(header.getSequence(refID).getSequenceName(), 1);
             // attempt to read 1st container must fail:
+            it.hasNext();
```

Comment:
We don't actually need this line at all anymore since it the advance now happens when the iterator is initialized.